### PR TITLE
feat(#273,#363,#278): Implements additional copy buttons, bugfixes

### LINKED ISSUE #273: Bug: Button for a drop-down menu is not working in Camel Operations
The button (3 dots) doesn't react on clicking. I assume the click on it should show a drop-down menu.

Screenshot 2023-05-04 at 13 53 54

### PR DESCRIPTION
**Pull request by joshiraez**
Implements #278 
Fixes #273 #363 

While fixing this I noticed I think there is some kind of bug in the contexts. Pages are being refreshed endlessly even when their state is not changed. Should I file a bug for this? I'm not sure if it's expected because the tree is constantly being refreshed.

### CODE REVIEW

**Review by tadayosi — CHANGES_REQUESTED** [review]
#363 should be already fixed with #360. Please revert the part which is supposed to fix #363.

IMO, `useMemo` or `useCallback` should be used only after we try our best to modularise React components with minimum states. The first approach should be always splitting components into parts so that where rerendering happens is narrowed.

File: packages/hawtio/src/plugins/shared/operations/OperationForm.tsx [hunk 1]
```diff
@@ -185,14 +182,57 @@ export const OperationForm: React.FunctionComponent<OperationFormProps> = props
     setIsDropdownOpen(!isDropdownOpen)
   }
 
+  const copyToClipboard = (text: string) => {
+    navigator.clipboard.writeText(text)
+    setCurrentAlerts(prev => [
+      ...prev,
+      <Alert key={`{alert-${prev.length + 1}`} isLiveRegion timeout={2000} title='Copied to clipboard' />,
```
Comment (tadayosi) [inline packages/hawtio/src/plugins/shared/operations/OperationForm.tsx:189]: Default alert timeout we use throughout hawtio-next is 8s (= 8000). Unless it was the behaviour from the old Hawtio, we should apply the same timeout duration here as well.

**Comment by joshiraez** [comment]
Oh great that it was already fixed. I also see that https://github.com/hawtio/hawtio-next/issues/346 you already noticed the frequent refreshes.

Ok, I'll refactor it to a different component instead of using useMemo. Let me a moment

File: packages/hawtio/src/plugins/shared/operations/OperationForm.tsx (hunk 1 above)
Comment (joshiraez) [reply packages/hawtio/src/plugins/shared/operations/OperationForm.tsx:189]: Alright. I checked the old-hawtio and 8 seconds seemed too long but I don't think there is any strong reason to not go with the default. It's patternfly's default as well so it'll make the code cleaner just removing that prop :smile:

**Comment by joshiraez** [comment]
The diff it's hard to understand. The fix has been to just move the OperationActions component outside of the component where it was, group it together with its own state and the new implemented functions in #278

File: packages/hawtio/src/plugins/shared/operations/OperationForm.tsx [hunk 2]
```diff
@@ -165,54 +167,47 @@ const OperationExecuteForm: React.FunctionComponent<{
...
+  }
 
-  const OperationActions = () => (
+  return (
     <DataListAction
       id={`operation-actions-${name}`}
       aria-label={`operation actions ${name}`}
       aria-labelledby={`${name} operation-actions-${name}`}
     >
+      <AlertGroup isToast isLiveRegion>
```
Comment (joshiraez) [inline packages/hawtio/src/plugins/shared/operations/OperationForm.tsx:208]: I just saw that we have an HawtIONotification element in the main page for this kind of toast alerts. Should I refactor this to use that better?

**Review by tadayosi — APPROVED** [review]

**Pull request merged**